src/actions.js

```diff
@@ -167,6 +167,13 @@ export const reserveTicket = ({ provider, personalInformation, ticket, ticketQua
                 promo_code: promoCode || null
             }));
 
+            // if the ticket is for myself, or have any attendee data, assign it to the ticket
```

Comment:
this is seems an overreaching
why use a boolean?
u only need to know if the ticket attendee_* needs to be filled or not